workshops/konami_code/README.md

```diff
@@ -0,0 +1,478 @@
+
+```js
+["KeyUp", "KeyDown", "KeyLeft", "KeyRight"]
+["KeyUp", "a", "KeyLeft", "z", "KeyDown"]
+["Shift", "Control", "Enter", "b"]
+["a", "a", "b", "b", "c", "d", "e", "KeyLeft"]
+```
+
+Next, we will make use of the confetti.js library which we have imported in our project. We will first declare some settings for the confetti. After we declare them, we will create a new `ConfettiGenerator` object which will help us implement the confetti on the site.
+
+```js
+const confettiSettings = { target: 'my-canvas', size: 1.4, max: 150 };
+const confetti = new ConfettiGenerator(confettiSettings);
+```
+**Explanation:** The settings for the confetti is an object. It is made with reference to [this](https://github.com/Agezao/confetti-js/) site. There are many more options which you can pass to the settings as you wish.
+
+The `target` attribute is used to specify the canvas with the id `my-canvas` on which the confetti will appear.
+
+The `size` attribute specifies the size for the confetti particles.
+
+The `max` attribute specifies the The number of props(confetti) to be rendered.
+
+The `ConfettiGenerator` contains a `render()` method and when it is called, we will be able to see confetti on our page.
+
+For more information on these attributes, [check here](https://github.com/Agezao/confetti-js/).
+
+Your code so far:
+```js
+const text = document.querySelector(".text");
+const secretDiv = document.querySelector('.secret-div');
+
+const keys = [];
+const secretCode = ["h", "a", "c", "k"];
+
+const confettiSettings = { target: 'my-canvas', size: 1.4, max: 150 };
+const confetti = new ConfettiGenerator(confettiSettings);
+```
+
+Next we will implement a function to check which keys are pressed by the user and when the keys match our Konami Code, it will get triggered.
+
+The function will take an 'event' attribute which will give us the key pressed by the user.
+
+```js
+function keysDetector(e) {
+  console.log(e.key);
+}
+```
+
+Outside the function, we will add an event listener on the window which will listen for all the `keyup` events and then trigger a particular function. It is written as follows:
+
+```js
+window.addEventListener('keyup', keysDetector);
+```
+
+**Explanation:** `addEventListener()` takes the first parameter as the event of which we need to keep track of. In our case it is the `keyup` event. The second parameter is a function which will get called whenever the event specified to it is performed. So whenever the `keyup` event is performed, the `keysDetector` function gets called.
+
+Learn more about [`addEventListener()`](https://www.w3schools.com/js/js_htmldom_eventlistener.asp).
+
+'RUN' this code, click on the preview window once so that it gets focused and then try typing any keys. Keep a watch on the console!
+
+Voila! The keys pressed by the user is getting logged! We will make use of this keys to check whether the keys matches the Konami Code.
```

Comment:
```suggestion
Learn more about [`addEventListener()`](https://www.w3schools.com/js/js_htmldom_eventlistener.asp).

'RUN' this code, click on the preview window once so that it gets focused and then try typing any keys. Keep a watch on the console!

Voila! The keys pressed by the user is getting logged! We will make use of these keys to check whether the keys match the Konami Code.
```